RecoTracker/LST/plugins/alpaka/LSTModulesDevESProducer.cc

```diff
@@ -22,7 +22,7 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
       descriptions.addWithDefaultLabel(desc);
     }
 
-    std::unique_ptr<SDL::LSTESData<SDL::DevHost>> produce(TrackerRecoGeometryRecord const& iRecord) {
+    std::unique_ptr<SDL::LSTESData<DevHost>> produce(TrackerRecoGeometryRecord const& iRecord) {
```

Comment:
is `DevHost` defined in `ALPAKA_ACCELERATOR_NAMESPACE` or is it in `alpaka_common::DevHost`?
For the latter, I don't see any `using namespace alpaka_common` in this file

Reply:
`DevHost` is defined both in `ALPAKA_ACCELERATOR_NAMESPACE` and `alpaka_common`